firmware/spade/src/shared/audio/piano.h

```diff
@@ -6,7 +6,7 @@ float strtof(const char *restrict nptr, char **restrict endptr);
 
 // gaps in your audio? increase this
 #define SAMPLES_PER_BUFFER (256*8)
-#define SAMPLES_PER_SECOND 24000
+#define SAMPLES_PER_SECOND 12000
```

Comment:
why is there audio code changed here?

Reply:
Same thing as https://github.com/hackclub/sprig/pull/1526#discussion_r1531480526

Reply:
understood - this is very important then, as well, to fix before landing